examples/metrics/src/main.rs

```diff
@@ -86,27 +88,3 @@ fn main() -> Result<(), Box<dyn Error>> {
     });
     Ok(())
 }
-
-/// Our network behaviour.
-///
-/// For illustrative purposes, this includes the [`keep_alive::Behaviour`]) behaviour so the ping actually happen
-/// and can be observed via the metrics.
```

Comment:
See this comment!

In this example, we want to retain the `identify` and the `ping` behaviour, otherwise the metrics don't show anything interesting :)